### DISCUSSION TITLE
kenapa saya tidak bisa mengunggah berkas?

### DISCUSSION THREAD

**Discussion by lisaadty** [community/community#38592] [GitHub Education]
kenapa saya tidak bisa mengunggah berkas untuk membuktikan saya mahasiswa

**Comment by queenofcorgis** [comment]
Hi @lisaadty, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
